projects/kit/components/input-date-time/input-date-time.template.html

```diff
@@ -1,7 +1,7 @@
 <tui-hosted-dropdown
     class="hosted"
     [canOpen]="interactive"
-    [content]="dropdown"
+    [content]="$explicit(dropdown)"
```

Comment:
You can pass type to `polymorpheus` instead:
```html
<ng-template #dropdown="polymorpheus" [polymorpheus]="type">
```
and
```ts
readonly type!: TuiContextWithImplicit<TuiActiveZoneDirective>;
```

Reply:
fixed